pkg/kudoctl/cmd/convert/convert.go

```diff
@@ -0,0 +1,196 @@
+package convert
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	kudo "github.com/kudobuilder/kudo/pkg/apis/kudo/v1alpha1"
+	"github.com/kudobuilder/kudo/pkg/util/helm"
+)
+
+// Options are configuration options for this command
+type Options struct {
+	ChartImportPath string
+	OutputPath      string
+}
+
+// DefaultOptions are options with the defaults applied
+var DefaultOptions = &Options{}
+
+// Run runs the convert command
+func Run(cmd *cobra.Command, args []string, options *Options) error {
```

Comment:
I would prefer to move this logic out... either functions that will be called by the CLI or remove `cmd *corba.Command`.    IMO we have other reasons to generate package files AND we could and should be testing this without the need for a cli or e2e test.